### PR TITLE
Templates: fixes few typos and inconsistencies in content 

### LINKED ISSUE #8468: Templates: Minor inconsistencies and typos
## Bug Description

Several of the newer templates have some typos or things that could be cleaned up to make them clearer. These are just my suggestions.  :)

### Baking Bread Guide
- Slide 4: Update casing – either important words should always start with a capital letter ("Temperature and Yeast") or not ("To knead or not to knead"), not both
- Slide 5: I think "other crust" is supposed to be "outer crust"
- Slide 5: Broken sentence, how about "To ensure a good lift and a beautiful outer crust, you'll want to..."
- Slide 9: Change "require" to "requires": "...yeast bread requires some kneading."
- Slide 10: Some words missing – update to "Too long and you might have a sour-tasting loaf..."

### Trendy Winter Veggie
- Slide 2: Change "trend" to "trends": "One of the biggest winter veggie trends this year..."
- Slide 3: Change "are" to "is": It has a nutty flavor and is perfect when steamed!"
- Slide 4: Items 2 and 3 are the same; something else to add?

### Ways to Eat Avocado
- Slide 2: Add "and" to last clause of sentence: "...fatty acids, and are loaded with fiber."
- Slide 6: Change "condense" to "condensed": "condensed milk"

### Album Releases
- Slide 1: Consider removing "American record producer"? Just seems out of place
- Slide 4: Remove "junior"

### Pizzas in NYC
 - Slide 12: Capitalize "Restaurants" to match rest of slide

### Ultimate Comparison
- Slide 10: Change "Ffective" to "Effective"

### 12 Hours in Barcelona

- Slide 2: Update second item's alignment to paper image to match alignment of first and third items: https://jmp.sh/0WM4EKM+/slide-2-alignment.png
- Slide 3: Remove comma after "Gothic"
- Slide 9: Add a comma after "now" – "ready for some bites now, right?"
- Slide 11: Capitalize "east" in "East Berlin"

### Fitness Apps Ranked
- Slide 1: Change "Written" to "Written by"
- Slide 4: The text here is confusing – is it free or not? "For someone who's just getting into a daily workout routine, Daily Workouts Fitness is a solid choice. Unfortunately, it's not free – it's a bit pricier of an app at just under $10."
- Slide 10: Change "Squat" to "squat" to match casing on rest of slide

### Indoor Garden Oasis DIY
- Slide 1: Remove period from end of "Home gardening guide"

### Street Style On The Go
- Slide 1: Change date format to US to match other templates: July 23, 2020
- Slide 3: Remove duplicate sentence "New York, no strange to being on the go."

### Belly Fat Workout
- Slide 1: Change date format to US to match other templates: July 23, 2020

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

### PR DESCRIPTION
**Pull request by divyarajmasani**
## Context

Some of the newly added story templates have typos and inconsistencies in the content. A few suggestions were given in issue #8468 

## Summary

This PR simply updates the necessary template JSON with given suggestions in attempt to make the stories more clearer.

## User-facing changes

A few templates with slightly updated text content.

## Checklist

- [x] This PR addresses an existing issue and I have linked this PR to it in ZenHub
- [x] I have tested this code to the best of my abilities
- [x] I have verified accessibility to the best of my abilities ([docs](https://github.com/google/web-stories-wp/blob/main/docs/accessibility-testing.md))
- [x] I have verified i18n and l10n (translation, right-to-left layout) to the best of my abilities
- [x] This PR contains automated tests (unit, integration, and/or e2e) to verify the code works as intended ([docs](https://github.com/google/web-stories-wp/tree/main/docs#testing))
- [x] I have added documentation where necessary
- [x] I have added a matching `Type: XYZ` label to the PR

---

Fixes #8468

### CODE REVIEW

File: packages/templates/src/raw/fitness-apps-ranked/template.json [hunk 1]
```diff
@@ -1082,7 +1082,7 @@
           "lineHeight": 1.3,
           "textAlign": "initial",
           "padding": { "locked": true, "hasHiddenPadding": false },
-          "content": "<span style=\"font-weight: 700; color: #f1ff51\">For someone who’s just getting into a daily workout Daily Workouts Fitness is a solid-and-free -choice.</span>",
+          "content": "<span style=\"font-weight: 700; color: #f1ff51\">For someone who’s just getting into a daily workout Daily Workouts FitFrame is a solid-and-free-choice.</span>",
```
Comment (swissspidy) [inline packages/templates/src/raw/fitness-apps-ranked/template.json]: FitFrame?

File: packages/templates/src/raw/fitness-apps-ranked/template.json [hunk 2]
```diff
@@ -3625,7 +3550,7 @@
           "textAlign": "initial",
           "padding": { "locked": true, "hasHiddenPadding": false },
           "type": "text",
-          "content": "<span style=\"font-weight: 500; color: #0f2c41\">Learn how to do a Squat Correctly</span>",
+          "content": "<span style=\"font-weight: 500; color: #0f2c41\">Learn how to do a squat Correctly</span>",
```
Comment (swissspidy) [inline packages/templates/src/raw/fitness-apps-ranked/template.json]: Lowercase `correctly` too? Looks odd otherwise.

Comment (divyarajmasani) [reply packages/templates/src/raw/fitness-apps-ranked/template.json]: Agree, this is still under review, with ongoing changes. Should be ready for your review by Tuesday evening.

Sorry to have raised false alarm 😅

File: packages/templates/src/raw/fitness-apps-ranked/template.json [hunk 3]
```diff
@@ -1082,7 +1082,7 @@
           "lineHeight": 1.3,
           "textAlign": "initial",
           "padding": { "locked": true, "hasHiddenPadding": false },
-          "content": "<span style=\"font-weight: 700; color: #f1ff51\">For someone who’s just getting into a daily workout Daily Workouts Fitness is a solid-and-free -choice.</span>",
+          "content": "<span style=\"font-weight: 700; color: #f1ff51\">For someone who’s just getting into a daily workout FitFrame is a solid-and-free-choice.</span>",
```
Comment (maitreyie-chavan) [inline packages/templates/src/raw/fitness-apps-ranked/template.json]: Add a comma after `daily workout` and remove the hyphen before `choice`

**Final sentence** - For someone who’s just getting into a daily workout, FitFrame is a solid-and-free choice.

Comment (divyarajmasani) [reply packages/templates/src/raw/fitness-apps-ranked/template.json]: updated.

**Review by swissspidy — APPROVED** [review]

**Pull request merged**